decidim-core/app/helpers/decidim/application_helper.rb

```diff
@@ -40,18 +40,42 @@ def present(object, presenter_class: nil)
     # can easily manage data without having to look for it at the admin
     # panel when they're at a public page.
     #
-    # link          - The String with the URL.
+    # link_url      - The String with the URL.
     # action        - The Symbol action to check the permissions for.
     # subject       - The Symbol subject to perform the action to.
     # extra_context - An optional Hash to check the permissions.
     #
     # Returns nothing.
-    def edit_link(link, action, subject, extra_context = {})
+    def edit_link(link_url, action, subject, extra_context = {}, link_options = { class: "topbar__edit__link" })
```

Comment:
Can you update the docs to explain the new argument, please? 😄

Reply:
Sure, I'll update the docs